src/app/cube/content-pages/contribute-page/copy.ts

```diff
@@ -0,0 +1,162 @@
+                        'primary way to begin designing a learning object. All draft (non-released) and released learning objects can be' +
+                        ' viewed within the dashboard. In addition, a status designates all learning objects while on the dashboard.'
+                    },
+                    {
+                        question: 'The Learning Object Builder',
+                        answer: 'When clicking the <em>"NEW +"</em> button, it will direct you to the learning object builder to start ' +
+                        'designing a learning object. The learning object builder is where you will enter your curriculum for ' +
+                        'submitting on CLARK. All required information is listed throughout the builder through its three sections: ' +
+                        '<em>Basic Information</em>, <em>Learning Outcomes</em>, and <em>Materials</em>. For more details, please see ' +
+                        'the <em>"Creating Learning Objects"</em> tab.'
+                    }
+                ]
+            }
+        ]
+    },
+    creating: {
+        title: 'Creating Learning Objects',
+        topics: [
+            {
+                title: 'Building a Learning Object',
+                questions: [
+                    {
+                        question: 'Basic Information',
+                        answer: 'The basic information tab covers the required descriptive tags of a learning object. These tags include' +
+                        ' the learning object name, known authors/contributors, learning object length, designated academic levels,' +
+                        ' and learning object description.'
+                    },
+                    {
+                        question: 'Learning Outcomes',
+                        answer:
+                        `
+                        <p>
+                        The learning outcomes tab covers the set of outcomes a user will accomplish after completing the 
+                        learning object. An outcome is a statement made by the author through the built-in Blooming Onion Builder, 
+                        which uses a provided list of action verbs tied to an academic goal. An example of this is, "Explain why ethics 
+                        is important in cybersecurity." 
+                        </p>
+                        <p>
+                        The <em>Blooming Onion Builder</em> also allows authors to map out their outcomes to 
+                        standardized curricular guidelines. A curricular guideline is a specific instructional statement created by a 
+                        recognized academic-adjacent group that focuses on a particular academic purpose. An example is applying the 
+                        <em>CAE Cyber Defense - Cybersecurity Ethics (KU3) - 2019</em> curricular guideline to our ethics outcome. This 
```

Comment:
Again, is this reference a released learning object?

Reply:
Again for this, it does not reference a learning object. It is a made up example using current curricular guidelines to map to a made up learning outcome.